citi-onboarding/acervusej-onepage
# Feat/contact

Things I've done:

- Create the Contact Us section responsively
- Create the model, controller and route for the Reasons of Contact
- Get the information for each Reasons of Contact item registered in the database, via the back-end route

How to test it:

- First download the node modules on the client and the server. If you have already done that, then:
- Change the values on .env.
- Go to client on the terminal (cd client) and put npm run start.
- On the browser write http://localhost:3000/.
- See if the contact us works both in mobile and desktop view.
- Look at the code to see if it has everything that it should have.
